### PR TITLE
Add io.github.seadve.Delineate

### PR DESCRIPTION
**Pull request by SeaDve**
### Please confirm your submission meets all the criteria

- [X] Please describe the application briefly.
  
Delineate provides facilities to edit and draw graphs specified in the [DOT language](https://graphviz.org/doc/info/lang.html). It is designed to be a simple and intuitive tool for creating and editing graphs, with a focus on the user experience.

- [X] The domain used for the application ID is [controlled by the application developers][appid-domain] and the [application id guidelines][appid] are followed.
- [X] I have read the and followed all the [Submission and licence requirements][reqs].
- [X] I have [built][build] and tested the submission locally.
- [X] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about this submission. **Link:** https://github.com/SeaDve/Delineate

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

File: io.github.seadve.Delineate.json [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name": "icu74",
```
Comment (hfiguiere) [inline io.github.seadve.Delineate.json:33]: there is libicu75 in the runtime.

**Comment by hfiguiere** [comment]
bot, build io.github.seadve.Delineate

File: io.github.seadve.Delineate.json (hunk 1 above)
Comment (SeaDve) [reply io.github.seadve.Delineate.json]: Thanks, nice to know!

Amended the commit.

**Comment by SeaDve** [comment]
> Build [148681 successful](https://buildbot.flathub.org/#/builders/6/builds/148681) To test this build, install it from the testing repository:
> 
> ```
> flatpak install --user https://dl.flathub.org/build-repo/131766/io.github.seadve.Delineate.flatpakref
> ```

Tested and everything seems to work well.

File: io.github.seadve.Delineate.json [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
```
Comment (TingPing) [inline io.github.seadve.Delineate.json:29]: Its harmless, but this seems to be a copy paste of directories that have nothing to do with this package.

Comment (SeaDve) [reply io.github.seadve.Delineate.json:29]: Ooh, I copied with other Rust app I published. I'm gonna check.

Comment (SeaDve) [reply io.github.seadve.Delineate.json]: Amended and it seems those files/directories do not actually exist in the build, as checked on the `builddir` directory from the following command:

```bash
flatpak run org.flatpak.Builder --force-clean --sandbox --user --install --install-deps-from=flathub --ccache --mirror-screenshots-url=https://dl.flathub.org/media/ --repo=repo builddir <manifest>
```

**Pull request closed without merging**